user-guide/components/index.njk

```diff
@@ -39,14 +39,18 @@ title: 13· Components
 <p>If a file is using a component from a Shared library that has been updated, a notification will be shown offering the options for update or dismiss.</p>
 <p><img src="/img/components-updatenotification.png" alt="components" /></p>
 
-<h3 id="component-overrides">Component overrides</h3>
+<h3 id="component-overrides">Component Overrides</h3>
 <p>Overrides are differences at the copy components from their main components. With overrides you can keep changes at the component copies while maintaining synchronization with the Main component.</p>
 <p>Component layers with overrides are marked with a “*” at the layers list.</p>
 <p><img src="/img/components-overrides.gif" alt="components" /></p>
 <h4>Reset overrides</h4>
 <p>Right click and select the option “Reset overrides” at the component menu to get it to the state of the Main component.</p>
 
-<h3 id="component-detach">Detach instance</h3>
+<p><strong>Note:</strong> The displayed layer name and it's contents are different things. For example,
+changing the layer name for a text layer does not change what is displayed in the viewport. This
+may cause some confusion when using components and text layers with overrides.
```

Comment:
Sorry, I don't understand this. For example I may have a label for a button, the layer name is "Text 1" and the contents are "Submit". Those are quite different things, indeed. How could anyone have confusion with them? Could you give me an example?